app/common/ApiConstants.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 HM Revenue & Customs
+ * Copyright 2022 HM Revenue & Customs
```

Comment:
There are a few files still with 2021 header, including the scalastyle config which checks the header is as expected.

Can you change the remaining headers and run scalastyle to make sure there are no "Header does not match expected text" warnings

Reply:
Sorry just seen this, I will sort that yeah